### ISSUE TITLE
jquery rating version displays NULL

### ISSUE THREAD

**Issue by nelero** [surveyjs/surveyjs#238]
if i use the latest version of the jquery survey with a rating question, it displays null for near min and max values
![capture_jquery_rating](https://cloud.githubusercontent.com/assets/593714/23017188/9c94be04-f439-11e6-9be8-dea48aff6700.PNG)

JSON : 
```
{
 pages: [
  {
   name: "page1",
   questions: [
    {
     type: "rating",
     name: "global_rating",
     rateValues: [
      "1",
      "2",
      "3",
      "4",
      "5"
     ],
     title: "Rate from 1 to 5 your satisfaction"
    },
]
}
]
}
```

**Comment by andrewtelnov** [comment]
@nelero Yes, indeed there is a bug. I will work on the fix. As a workaround, please set the following properties:
```
     maximumRateDescription: " ",
     mininumRateDescription: " ",

```
JSON:

```
{
 pages: [
  {
   name: "page1",
   questions: [
    {
     type: "rating",
     maximumRateDescription: " ",
     mininumRateDescription: " ",
     name: "global_rating",
     title: "Rate from 1 to 5 your satisfaction",
     rateValues: [
      "1",
      "2",
      "3",
      "4",
      "5"
     ]
    }
   ]
  }
 ]
}
```

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
Fixed in v0.12.0
